src/app/modals/modals-helper.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { Subject } from 'rxjs/Subject';
+import { Log } from 'ng2-logger';
+
+import { MatDialog, MatDialogRef } from '@angular/material';
+import { Observable } from 'rxjs/Observable';
+
+import { RpcStateService } from 'app/core/rpc/rpc-state/rpc-state.service';
+
+import { UnlockwalletComponent } from 'app/modals/unlockwallet/unlockwallet.component';
+import { ModalsConfig } from './models/modals.config.interface';
```

Comment:
move ModalsConfig in here perhaps? Or are you going to make more models?

Reply:
ahh I see why - no prob.

Reply:
thanks :)